check50/internal.py

```diff
@@ -13,12 +13,19 @@
 #: Directory containing the check and its associated files
 check_dir = None
 
-#: Temporary directory in which check is being run
+#: Temporary directory in which the current check is being run
 run_dir = None
 
+#: Temporary directory that is the root (parent) of all run_dir(s)
```

Comment:
Why is this necessary instead of just using run_dir.parent?

Reply:
`run_dir` only gets set inside a check, that is after having imported the checks module. Whereas `run_root_dir` gets set before importing. This makes it possible to store things prior to running checks.